src/design-guidelines/action-buttons/action-buttons.md

```diff
@@ -0,0 +1,121 @@
+# Action buttons
+
+Action buttons are typically used in dialogs' footers, or in headers. These are buttons indicate to users that they must deliberately choose one action from a set of actions which are presented, for the system to continue or finish a task.
```

Comment:
I don't think _dialog's footer_ (or "dialogs' footers" in this case is grammatically wrong 🙃. Maybe "dialogs' footers" sounds a bit silly though. 
![image](https://user-images.githubusercontent.com/35954987/126750629-90f57933-1781-4859-a29c-edad1f7a3dc7.png)
[form here](https://developer.apple.com/design/human-interface-guidelines/macos/buttons/bevel-buttons/).

Reply:
No, it's not that it's grammatically wrong. It's just that "used in dialog footers" works just as well and is less clunky to say. The difference is easier to explain in Swedish:

- used in dialogs' footers -> används i dialogers sidfötter
- used in a dialog's footer -> används i en dialogs sidfot
- used in dialog footers -> används i dialogsidfötter

This is just to explain the relation between the different forms. In Swedish, I think the last form is the clunkiest, but in English, I think it's very neat.

"Used in a window's main content area" can't really be rewritten in the same way, because while 
(at least I think) you can call the footer of a dialog a "dialog footer", it doesn't really work to call the main content area of a window the "window main content area" (think of it as a single concatenated word in Swedish, like "windowmaincontentarea", vs "dialogfooter").

See the use of "dialog body" and "dialog footer" here:
![image](https://user-images.githubusercontent.com/154725/126753940-e61b3194-6b09-40f2-abf9-2401ae71557f.png)

Reply:
(You're perfectly free to choose which form to use of course, it's just a suggestion 😊)